migration-guide/business-central.adoc

```diff
@@ -0,0 +1,150 @@
+include::../partials/attributes.adoc[]
+= Migrating from Business Central to IBM {DEV_TOOLS}
+
+== Overview
+
+Business Central was a web-based platform for authoring business automation assets such as data objects, rules, processes, cases, and planning entities. 
+It included capabilities for monitoring, administration, and management of business automation assets.  With the release of both IBM {DEV_TOOLS} and IBM {CANVAS} 
+most of the asset management capabilities were replaced or sometimes take a different approach altogether.  This section describes 'the' migration of business 
+assets 'that are' created in Business Central so that they can be maintained by developers by using either {DEV_TOOLS} or {CANVAS} or both.
+== Prerequisites
+
+The following tools are required to migrate from Business Central to {DEV_TOOLS}. It is assumed that you have a current version of Business Central running to export 
+certain types of assets to their supported form in {PRODUCT_SHORT}:
+
+include::../partials/prerequisites.adoc[]
+
+== Business Automation Assets in Businss Central
+
+The following types of business automation assets were available within Business Central with a focus on the the assets that are deprecated in Version 9.0 too. 
```

Comment:
Not sure if "deprecated" is the correct term here. By definition, a deprecated feature is still available, but it's highly recommended to don't use it because it will be removed soon. In our case, the involved features are already removed and not deprecated.

Reply:
... assets are no longer supported in version 9.0.

How does that sound?

Reply:
agreed, I like that better